src/classes/OPP_OpportunityNaming_TEST.cls

```diff
@@ -276,4 +278,207 @@ private class OPP_OpportunityNaming_TEST
         System.assertEquals(1, errors.size(), 'An error record should be created.');
 
     }
+
+    /*******************************************************************************************************
+    * @description Verify if opportunity naming format works when creating opportunity using a user with
+    * language other than English.
+    ********************************************************************************************************/
+    @isTest
+    private static void shouldFormatOppNameWhenInsertingForNonEnglishUser() {
+
+        Opportunity_Naming_Settings__c ons = new Opportunity_Naming_Settings__c(
```

Comment:
Let's create a helper method that will insert and refresh the Opp naming settings and reuse throughout new tests.